client/app/components/tagging/tagging.directive.js

```diff
@@ -144,8 +144,8 @@
         return {id: tag.id,
                 category: {id: tag.category.id},
                 categorization: {
-                  display_name: tag.categorization.display_name,
-                }
+                  displayName: tag.categorization.display_name,
+                },
```

Comment:
@AllenBW To resolve this, in client/app/states/designer/blueprints/editor/editor.state.js

<pre>
function StateController($log, blueprint, blueprintTags, serviceTemplates) {
    var vm = this;
    vm.title = 'Blueprint Designer';
    if (blueprint) {
      if (blueprintTags && blueprintTags.resources) {
        blueprint.tags = blueprintTags.resources;
        **angular.forEach(blueprint.tags, processTag);   // <--- Add this line**
      }
      vm.blueprint = blueprint;
    }
    **// Add this function
    function processTag(tag) {
      tag.categorization.displayName = tag.categorization.display_name;
      delete(tag.categorization.display_name);
    }**
<pre>